VISION.md

```diff
@@ -0,0 +1,34 @@
+# The Vision
+
+This page describes the main principles that drive the development of `probatus` as well as the general directions, in which the development of the package will be heading.
+
+## The Purpose
+
+`probatus` has started as a side project of Data Scientists at ING Bank. 
+It consists of tools that help in analysing binary classification models and the data used to develop them.
+We have decided to open-source it, in order to share the tools and enable collaboration with the Data Science community.
+
+Our main goals are:
+
+- Continue maintaining the tools that we have built, and make sure that they are well documented and tested
+- Continuously extend functionality available in the package
+- Build a community of users, which use the package in day-to-day work and learn from each other, while contributing to probatus
```

Comment:
Sound good, but could be more specific. Can we provide any more guidance on the scoping of the package other than 'tools to help analyze binary classifcation models and data used to train them' ?